tyr/servers/mongo/node.py

```diff
@@ -24,10 +25,17 @@ def __init__(self, group = None, server_type = None, instance_type = None,
                                         environment, ami, region, role,
                                         keypair, availability_zone,
                                         security_groups, block_devices,
-                                        chef_path)
+                                        chef_path, subnet_id)
 
     def configure(self):
 
+        self.security_groups = [
+            'management',
+            'chef-nodes',
+            '{envcl}'.format(envcl=self.envcl),
```

Comment:
If the security group is just the value of `self.envcl`, you could just use `self.envcl` instead of formatting a string with that value.